pkgs/development/libraries/openmpi/default.nix

```diff
@@ -14,6 +14,8 @@
 # Enable libfabric support (necessary for Omnipath networks) on x86_64 linux
 , fabricSupport ? stdenv.isLinux && stdenv.isx86_64
 
+# enable C++ bindings
+, enableCpp ? false
```

Comment:
We may not need to make it conditional but it can always be enabled. I will run nixpkgs on it to see if that affects other packages.

Reply:
It is OK to enable it per default. Let's remove the conditional here and always enable it on Linux.

Reply:
Thanks for the review! I've made the change.